gym/wrappers/pixel_observation.py

```diff
@@ -2,6 +2,10 @@
 
 import collections
 import copy
+try:
+    from collections.abc import MutableMapping
+except ImportError:
+    from collections import MutableMapping
```

Comment:
```suggestion
from six.move.collections_abc import MutableMapping
```

Reply:
TIL, thanks for the tip. Made the changes.